misc/multiaddr/Cargo.toml

```diff
@@ -6,14 +6,14 @@ description = "Implementation of the multiaddr format"
 homepage = "https://github.com/libp2p/rust-libp2p"
 keywords = ["multiaddr", "ipfs"]
 license = "MIT"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
The `TryFrom<Bytes>` and `TryFrom<BytesMut>` impls have been removed, so maybe we should go for 0.7.0 here.